src/annotator/test/highlighter-test.js

```diff
@@ -6,6 +8,60 @@ import {
   getBoundingClientRect,
 } from '../highlighter';
 
+/**
+ * Preact component that renders a simplified version of the DOM structure
+ * of PDF.js pages.
+ *
+ * This is used to test PDF-specific highlighting behavior.
+ */
+function PdfPage() {
+  return (
+    <div className="page">
+      <div className="canvasWrapper">
+        {/* Canvas where PDF.js renders the visual PDF output. */}
+        <canvas />
+      </div>
+      {/* Transparent text layer created by PDF.js to enable text selection */}
+      <div className="textLayer">
+        {/* Text span created to correspond to some text rendered into the canvas.
+            Hypothesis creates `<hypothesis-highlight>` elements here. */}
+        <span className="testText">Text to highlight</span>
+      </div>
+    </div>
+  );
+}
+
+/**
+ * Highlight the text in a fake PDF page.
+ *
+ * @param {HTMLElement} - HTML element into which `PdfPage` component has been
+ *   rendered
+ * @return {HTMLElement} - `<hypothesis-highlight>` element
+ */
+function highlightPdfRange(pdfPage) {
```

Comment:
pdfPage is also the name of the function above, I would suggest a discrete name to avoid confusion.